src/components/dropdown-filter/dropdown-filter.js

```diff
@@ -510,6 +518,20 @@ class DropdownFilter extends Dropdown {
       'data-role': props['data-role']
     };
   }
+
+  onUpArrow = (list, element) => {
+    if (this.props.create) {
+      this.blockBlur = true;
+    }
+    return super.onUpArrow(list, element);
```

Comment:
would prefer to avoid overriding and calling super if possible, what is this change for?